src/phases/query/RepeatedPathTraversal.yml

```diff
@@ -0,0 +1,471 @@
+              {"a.b.e": 1},
+              {"a.b.f": 1},
+              {"a.b.g": 1}]}
+
+- Name: AggregationExpressionRepeatedPathTraversalInequality
+  Type: CrudActor
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [5]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {$expr:
+                       {$or: [
+                         {$ne: ["$a.b.c", "$a.b.d"]},
+                         {$ne: ["$a.b.d", "$a.b.e"]},
+                         {$ne: ["$a.b.e", "$a.b.f"]},
+                         {$ne: ["$a.b.f", "$a.b.g"]},
+                         {$ne: ["$a.b.g", "$a.b.c"]}]}}
+
+- Name: MatchExpressionRepeatedPathTraversalInequality
+  Type: CrudActor
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [6]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {$or: [{"a.b.c": {$ne: 0}},
+                           {"a.b.d": {$ne: 0}},
+                           {"a.b.e": {$ne: 0}},
+                           {"a.b.f": {$ne: 0}},
+                           {"a.b.g": {$ne: 0}}]}
+
+# Double the number of clauses.
+- Name: AggregationExpressionRepeatedPathTraversalWidePredicate
+  Type: CrudActor
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [7]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter:
+              {$expr: {$or: [
+                {$eq: ["$a.b.c", 1]},
+                {$eq: ["$a.b.d", 1]},
+                {$eq: ["$a.b.e", 1]},
+                {$eq: ["$a.b.f", 1]},
+                {$eq: ["$a.b.g", 1]},
+                {$eq: ["$a.b.h", 1]},
+                {$eq: ["$a.b.i", 1]},
+                {$eq: ["$a.b.j", 1]},
+                {$eq: ["$a.b.k", 1]},
+                {$eq: ["$a.b.l", 1]}]}}
+
+- Name: MatchExpressionRepeatedPathTraversalWidePredicate
+  Type: CrudActor
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [8]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {$or: [
+              {"a.b.c": 1},
+              {"a.b.d": 1},
+              {"a.b.e": 1},
+              {"a.b.f": 1},
+              {"a.b.g": 1},
+              {"a.b.h": 1},
+              {"a.b.i": 1},
+              {"a.b.j": 1},
+              {"a.b.k": 1},
+              {"a.b.l": 1}]}
+
+- ActorFromTemplate:
```

Comment:
All actors before this are the same as before (except `InsertRepeatedPathTraversalData`).